### PR TITLE
Remove "Default" location if multiple locations are set

### PR DESCRIPTION
**Pull request by AxeyOS**
Remove "Default" location if multiple locations are set

### CODE REVIEW

File: login.php [hunk 1]
```diff
@@ -55,11 +55,12 @@
 						<br/>
 						<li><a href="#" >Location</a> </li>
 						<select name="location" tabindex=3 class="generic" >
-							<option value="default">Default</option>
 							<?php
 								if (isset($configValues['CONFIG_LOCATIONS']) && is_array($configValues['CONFIG_LOCATIONS'])) {
```
Comment (MiguelVis) [inline login.php]: Nice PR. But in that case, I would add one more check to line 59:

if (isset($configValues['CONFIG_LOCATIONS']) && is_array($configValues['CONFIG_LOCATIONS']) **&& count($configValues['CONFIG_LOCATIONS']) > 0**) {

What do you think?

Comment (AxeyOS) [reply login.php]: Nice, it's gonna be good if the user leaves the array empty

Comment (AxeyOS) [reply login.php]: Added

**Comment by MiguelVis** [comment]
Thanks @AxeyOS.

**Pull request merged**